configs/ASRock/Z390-Extreme4.conf

```diff
@@ -0,0 +1,107 @@
+# ASRock Z3900 Extreme4
+# 2019, contributed by Alex Cherkayev <doctor64@gmail.com>
+#
+# dmi: board_name:   Z390 Extreme4
+# dmi: board_vendor: ASRock
+# dmi: bios_version: P4.00
+# cpu: Intel(R) Core(TM) i7-9700K CPU
+chip "nct6791-isa-*"
+
+    # Fans
+    label fan1 "CHA Fan 3"
+    label fan2 "CPU Fan 1"
+    label fan3 "CPU Fan 2/WP"
+    label fan4 "CHA Fan 1"
+    label fan5 "CHA Fan 2"
```

Comment:
Looking at the manual, all the chassis fans can alternatively be water pumps. Since you're highlighting the water pump possibility in "CPU Fan 2/WP", you might as well do the same for the chassis fans too.

Reply:
Fixed